src/main/scala/ru/org/codingteam/horta/plugins/pet/Extractors.scala

```diff
@@ -0,0 +1,19 @@
+package ru.org.codingteam.horta.plugins.pet
+
+object DyingPet {
+
+  def unapply(pet: PetData) = pet.health <= 0 || pet.satiation <= 0
```

Comment:
This is interesting. I always thought that `unapply` is supposed to return `Option[T]`.